examples/integrations/server.js

```diff
@@ -0,0 +1,74 @@
+const express = require("express");
+const { makeExecutableSchema } = require("graphql-tools");
+const queries = require("./queries");
+const axios = require("axios");
+const {
+  ApolloVoyagerServer,
+  gql
+} = require("../../packages/apollo-voyager-server");
+
+// Types
+const typeDefs = gql`
+  type CharacterInfo {
+    name: String
+    homeworld: String
+    gender: String
+  }
+  type Query {
+    getCharacterInfo(id: ID!): CharacterInfo
+    hello: String
+  }
+`;
+
+// Resolver functions. This is our business logic
+const resolvers = {
+  Query: {
+    getCharacterInfo: async (obj, args, context, info) => {
+      //gets JSON response from the external API, parameter `id` is passed to the external API
+      const response = await axios.get(
+        `http://swapi.co/api/people/${args.id}`,
```

Comment:
Missing default value can cause error. 
`https://swapi.co/api/people/undefined/` Add random fela to avoid this.